soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.nrf52811_QFAA

```diff
@@ -14,4 +14,8 @@ config NUM_IRQS
 config NET_CONFIG_IEEE802154_DEV_NAME
 	default IEEE802154_NRF5_DRV_NAME
 
+config IEEE802154_NRF5
+	default y
```

Comment:
Since you are moving this, can you take the opportunity to use the more compact syntax:

`default IEEE802154` and remove the line below?

Reply:
This syntax seems a little unreadable for me, probably because I'm not fluent in Kconfigs. How about `default y if IEEE802154`? It is also a oneliner, but seems much clearer.

Reply:
We've been settling on `default <dependency>` on all other files, and in any new Kconfig code, so I'd rather we use the same syntax here.